app/src/main/java/fr/free/nrw/commons/upload/depicts/DepictsPresenter.kt

```diff
@@ -100,14 +102,44 @@ class DepictsPresenter @Inject constructor(
      * Check if depictions were selected
      * from the depiction list
      */
-    override fun verifyDepictions() {
+    override fun verifyDepictions(application: Application) {
         if (repository.selectedDepictions.isNotEmpty()) {
+            if (application.applicationContext != null) {
```

Comment:
We have dagger in place for DB read write operations, we need not worry about the context, please refer other similar classes to see how we have used the corresponding DAO's to query the DB